### ISSUE TITLE
Import note to user's own folder in Zeppelin UI

### ISSUE THREAD

**Issue by NigelHambly** [wfau/aglais#848]
This may be a non-issue; if so, apologies for the noise.

The only way I can see to import an existing notebook into my user folder is via the import feature in Zeppelin which puts the note into the top-level folder. Then I simply rename it into my own folder. A bit clunky, but OK.

However this may work for me because nch has permission to write into the top of the folder hierarchy. But will it work for our general users? If not it may be impossible for them it import a locally saved notebook at all.

Please test independently with a test user account and get back to me.

**Comment by Zarquan** [comment]
I've had a quick look using a test deployment on red.
The upload dialog box itself is a little cumbersome to use. The user can specify a folder to import the note to, but it has to be done before they select the file to upload (see attached). Perhaps we need some user how-to manual pages on our wiki to explain things like this.
![upload-000](https://user-images.githubusercontent.com/4954609/178144005-9303220c-3b33-4cef-83b9-a2ba1ce59bd9.png)

**Comment by Zarquan** [comment]
However ... I also found that in the version that I'm using the permissions seem to allow **_anyone_** to upload or clone a new new note to **_any_** folder, including the top level folder. Created a new issue (#849 849) to cover this.

**Comment by NigelHambly** [comment]
Ah ok I missed the "Import note as..." functionality. That's fine for now. Thanks for opening the other issue. I'll close this as AFAIC the existing functionality is fine (if lacking a little idiot-proofness ... especially for idiots like me).
